packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/react/integrationsummary.md

```diff
@@ -0,0 +1,5 @@
+<div class="full">
+
+![Sequence diagram illustrating the password-optional sign-up use case for React.](/img/pwd-optional/pwd-optional-sign-up-js-react-summary.png)
```

Comment:
Corrections to flow diagram

* Chevron2 is missing a hyphen in "sign-up link"
* Arrow from chevron 2 is missing hypen "Click sign-up link"
* In grey box, "Generate sign-up page" is missing a hyphen

Reply:
Updated.